ipn/ipnlocal/c2n_pprof.go

```diff
@@ -10,6 +10,8 @@ import (
 	"runtime"
 	"runtime/pprof"
 	"strconv"
+
+	hpprof "tailscale.com/tempfork/pprof"
```

Comment:
actually I'd like to delete this whole forked package. It was only one caller left inside corp and it's redundant there now that we have c2n. Want to just do it by hand like in your earlier version and then also delete this package?

The net/http/pprof package is super minimal. It's not worth trying to reuse that code just to fight text/template and reflect.MethodByName coming along for the ride.